### ISSUE TITLE
Azure AD B2C Sign In - JAWS

### ISSUE THREAD

**Issue by sashiRachakonda** [azure-ad-b2c/samples#251]
[Azure AD B2C Sign In - Tab stops.pdf](https://github.com/azure-ad-b2c/samples/files/6743655/Azure.AD.B2C.Sign.In.-.Tab.stops.pdf)

1. On login page
2. Tab through the page to the "Sign up now" link
3. Note JAWS does not read the "Don't have an account?" when you tab through the page

JAWS does not read the "Don't have an account?" text on the login page.
The corresponding 'Don't Have an Account?' text was not read by JAWS.

**Comment by JasSuri** [comment]
This is not an issue with any sample here, rather the service. Better to raise a support ticket with Microsoft.
